build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -19,47 +19,31 @@ metadata:
     alm-examples: |-
       [
         {
-          "apiVersion": "maistra.io/v1",
+          "apiVersion": "maistra.io/v2",
           "kind": "ServiceMeshControlPlane",
           "metadata": {
-            "name": "basic-install",
+            "name": "basic",
             "namespace": "control-plane-namespace"
           },
           "spec": {
             "version": "v2.0",
```

Comment:
This is the field that I was seeing the error for.

Reply:
Yeah, but this change isn't what fixes it. The fact that we had a v1 SMCP in `alm-examples` meant that no field was actually initialized. This caused the name field to be set to "example", but there's a bug in the UI that makes it appear as though this is actually enter in the field, when it's actually not. So when you submit the form, there is no name set, hence the error.

Anyway, the change to v2 fixes it.

Reply:
Thanks. Do we need the default in the profiles as well? If you install via CLI, you don't get any of this, correct?